airflow/providers/google/cloud/transfers/sftp_to_gcs.py

```diff
@@ -93,6 +94,7 @@ def __init__(
         move_object: bool = False,
         impersonation_chain: str | Sequence[str] | None = None,
         sftp_prefetch: bool = True,
+        fail_on_sftp_file_not_exist: bool = True,
```

Comment:
Could you please shorten the parameter name?
Maybe something like `fail_if_not_exists`?